ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -468,67 +477,87 @@ KEXT_STATIC int HandleFileOpOperation(
         
         putCurrentVnode = true;
         
-        VirtualizationRootHandle root = RootHandle_None;
+        VirtualizationRootHandle toRoot = RootHandle_None, fromRoot = RootHandle_None;
         FsidInode vnodeFsidInode = {};
-        int pid;
-        if (!ShouldHandleFileOpEvent(
+        int pid = -1;
```

Comment:
why are we adding the -1 assignment now?

Reply:
I can't quite spot the case I was thinking of specifically at the time right now but it was to avoid undefined behaviour when reading the variable and we hadn't written the PID to it.

Reply:
I've looked over this now, and I'm not sure what the specific trigger was, possibly a complaint by the static analysis step, or possibly a debug output statement that printed `pid` in cases where it might not have been set.

In any case, passing around pointers or references to variables that haven't been initialised or may not have been initialised is something I consider something of a code smell, it's just really hard to reason about and explicitly initialising it costs almost nothing.